discuss/Formats.md

```diff
@@ -53,16 +53,23 @@ JSON is a fairly minimal format that lets you deliver sets of labeled data.  The
 ```JSON
 {
   "metadata": {
-    "lab":"Kerr",
-    "person":"Kerr",
+    "lab":{"location":"Janelia Research Campus", "name":"Kerr Lab" },
+    "who":"Kerr",
     "strain":"N2",
-    "date":"2015-04-27T15:15:32",
-    "units":"pixels"
+    "timestamp":"2015-04-27T15:15:32"
```

Comment:
I wasn't at Janelia at this time.  If we're going to pick something plausible, we could make it 2011.  Or we can change to a fictitious person.  (I don't want to make people at UC San Francisco or Calico feel that I'm not appreciating the opportunity I have had / have to work there.)